srv/components/system/firewall/install.sls

```diff
@@ -28,11 +28,6 @@ Unmask Firewall service:
       - Install firewalld
 
 # Ensure ssh works when the firwall servcie starts for the next time
-{% if 'mgmt0' in grains['ip4_interfaces'] and grains['ip4_interfaces']['mgmt0'] %}
```

Comment:
Are we sure to delete this one?

Reply:
Yes, it is the network teaming format that we would require later for 2 network setup.

Reply:
Yes, as we are not using it anywhere.
The configuration here is addressing global zone and not targeted ones.